virtual_rainforest/models/animals/scaling_functions.py

```diff
@@ -0,0 +1,124 @@
+"""The `models.animals.scaling_functions` module contains a set of functions containing
+scaling equations" (relationships between body-mass and a train) required by the broader
+:mod:`~virtual_rainforest.models.animals` module
+"""  # noqa: D205, D415
+
+from math import ceil
+
+from virtual_rainforest.models.animals.constants import (
+    DamuthsLawTerms,
+    FatMassTerms,
+    IntakeRateTerms,
+    MeatEnergy,
+    MetabolicRateTerms,
+    MuscleMassTerms,
+)
+
+
+def DamuthsLaw(mass: float) -> int:
```

Comment:
Functions should be lower_case

```suggestion
def damuths_law(mass: float) -> int:
```

And the same with all the other functions in this file.